src/main/java/me/jellysquid/mods/sodium/client/world/biome/BiomeColorCache.java

```diff
@@ -32,18 +36,23 @@ public BiomeColorCache(ColorResolver resolver, WorldSlice slice) {
 
         this.dim = 16 + ((this.radius + 2) * 2);
 
+        this.blendedColorsMinX = origin.getMinX() - 2;
+        this.blendedColorsMinZ = origin.getMinZ() - 2;
+
+        this.blendedColorsDim = 16 + 2 * 2;
```

Comment:
`blendedColorsDim` should be moved to a static variable (such as `BLENDED_ARRAY_WIDTH`) since it's constant. This should also allow the JVM to better optimize the `getBlendedColor` call.